callisto_core/delivery/forms.py

```diff
@@ -59,7 +59,7 @@ class ReportCreateForm(
     ReportBaseForm,
 ):
     message_confirmation_error = "passphrase and passphrase confirmation must match"
-    key_confirmation = fields.PassphraseField(label='Confirm Passphrase')
+    key_confirmation = fields.PassphraseField(label='Confirm Passphrase', label_suffix='*')
```

Comment:
`PassphraseField` is a class defined locally, add `label_suffix='*'` to its `__init__` https://github.com/project-callisto/callisto-core/blob/master/callisto_core/delivery/fields.py

Reply:
agreed. i applied this change.